hw3/style-hw3.css

```diff
@@ -0,0 +1,457 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	background-color: #fff;
+	min-width: 460px;
+}
+
+.banner__button_button,
+.subscription__submit_blue,
+.main-footer__button_blue {
+	background: linear-gradient(to top, #4290c1, #60a9d4);
+	cursor: pointer;
+	color: #fff;
+	border: none;
+	border-radius: 4px;
+}
+
+@font-face {
+    font-family: Dunkirk;
+    src: url(Dunkirk.otf);
+}
+
+/*------header-----*/
+.main-header {
+	min-height: 70px;
+	padding: 0 10%;
+	background: url(image/bcg.png) round;
```

Comment:
nope, use like one iterable part of the image (like square 5*5px) and use repeat for backgroung. It will reduce the size of request.